x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/index.tsx

```diff
@@ -203,13 +204,13 @@ export const EditOutputFlyout: React.FunctionComponent<EditOutputFlyoutProps> =
               helpText={
                 <FormattedMessage
                   id="xpack.fleet.settings.editOutputFlyout.logstashHostsInputDescription"
-                  defaultMessage="Specify the URLs that your agents will use to connect to a Logstash. For more information, see the {guideLink}."
+                  defaultMessage="Specify the addresses that your agents will use to connect to Logstash. {guideLink}."
```

Comment:
Does "address" instead of "URL" work? I feel it's less specific and may or may not include the protocol.

Reply:
It seems correct to me